README.md

```diff
@@ -90,6 +92,10 @@ Installing Dispatch depends on having a Kubernetes cluster with the Knative comp
 
 4. Deploy via helm chart (if helm is not installed and initialized, do that first):
     ```bash
+    cd charts/dispatch
+    helm dependency update
+    helm init
```

Comment:
I would probably add a new optional step above for `helm init` and also mentions that tiller requires cluster admin privileges (refer the legacy installation steps)

Reply:
Nit: `helm dep up` instead of the verbose command